pkg/pillar/hypervisor/kvm.go

```diff
@@ -158,6 +158,22 @@ const qemuConfTemplate = `# This file is automatically generated by domainmgr
   chardev = "charserial0"
   name = "org.lfedge.eve.console.0"
 
+{{if .DomainConfig.IsOCIContainer}}
```

Comment:
These additions might be causing the unit test failure; does test need to be updated?

Reply:
Yes, fixed that. Thanks.